Move stuff around for gnulib inclusion (need to check if this breaks
ass-needed stuff...)

--- Makefile
+++ Makefile
@@ -57,13 +57,13 @@
 ifeq ($(subst s,,$(MAKEFLAGS)),$(MAKEFLAGS))
 	@echo $(foreach a,$(APPLETS), : $a) :
 ifndef V
-	@echo $(CC) $(CFLAGS) $(PYFLAGS) $(LDFLAGS) main.c -o q $(LIBADD)
+	@echo $(CC) $(CFLAGS) $(PYFLAGS) main.c -o q $(LIBADD) $(LDFLAGS)
 endif
 endif
 ifndef V
-	@$(CC) $(WFLAGS) $(PYFLAGS) $(LDFLAGS) $(CFLAGS) $(HFLAGS) main.c -o q $(LIBADD)
+	@$(CC) $(WFLAGS) $(PYFLAGS) $(CFLAGS) $(HFLAGS) main.c -o q $(LIBADD) $(LDFLAGS)
 else
-	$(CC) $(WFLAGS) $(PYFLAGS) $(LDFLAGS) $(CFLAGS) $(HFLAGS) main.c -o q $(LIBADD)
+	$(CC) $(WFLAGS) $(PYFLAGS) $(CFLAGS) $(HFLAGS) main.c -o q $(LIBADD) $(LDFLAGS)
 endif
 
 .depend: $(SRC)
